fastlane/Fastfile

```diff
@@ -38,80 +68,182 @@ platform :ios do
 
   end
 
-  # Build and publish the Alpha version to TestFlight
+  # Build and publish version to TestFlight
```

Comment:
Not sure. All the comments are written this way, and I'm not sure it should be different. I also tend to add the `s`, but if you check the link on top of the document it's not done this way. Let me check.